### PR TITLE
Feature/ch166/create patient registration page

### PR DESCRIPTION
**Pull request by fahadhussain2**

### CODE REVIEW

File: packages/app/src/containers/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export * from './LandingPage';
+export * from './SignupPage';
+export * from './RegistrationDetails';
```
Comment (petermsouzajr) [inline packages/app/src/containers/index.ts:3]: i get a build failure on this line is there a file called `/RegistrationDetails`
`Unable to resolve "./RegistrationDetails" from "src/containers/index.ts"
Failed building JavaScript bundle.`

Comment (fahadhussain2) [reply packages/app/src/containers/index.ts]: @petermsouzajr This issue is resolved

**Comment by petermsouzajr** [comment]
@fahadhussain2 branch is 3 commits behind master, please update branch so it is current with master. please resolve branch conflicts as well

**Comment by fahadhussain2** [comment]
> master is three commits behind, please resolve conflicts

The conflicts are resolved and also the branch is updated with master @petermsouzajr

File: packages/app/src/containers/SignupPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        super(props);
+
+        this.state = {
+            email: '',
+            password: '',
+        };
+    }
+
+
+    public componentDidMount() {
```
Comment (petermsouzajr) [inline packages/app/src/containers/SignupPage/index.tsx:32]: is this being used?

Comment (fahadhussain2) [reply packages/app/src/containers/SignupPage/index.tsx:32]: @petermsouzajr We do not need it now but we may need this lifecycle method in the future so I just defined this method. I will remove this if you want.

**Pull request merged**